### ISSUE TITLE
user should be able to download zip file of packages and their dependencies

### ISSUE THREAD

**Issue by nswitanek** [teradata-aster-field/toaster#17]
For isolated Aster clusters where the DBA doesn't give users install file permissions, would be useful for users to be able to give vector of package names and get zip file of packages and dependencies to give to DBA for installation.

**Comment by briankreeger** [comment]
Is this a suggestion that we would have to keep track of all updated dependent packages?

**Comment by nswitanek** [comment]
No. Use case is R user running R in Aster on cluster not connected to Internet. Wants to use a certain package or set of packages. Those packages have dependencies. This function returns a zip file of source R packages for user to install on Aster herself or else hand off to DBA if she's restricted from installing in the cluster directly. Make sense? Have you seen similar or related use cases?

**Comment by briankreeger** [comment]
For me personally, I install all possible packages into a separate directory and can move that around to different machines.  

The following option needs to be set in .Rprofile:
# Set library for packages
.libPaths("/Users/bgk/rdeposit")

A It makes consistent environments easier to maintain.  But I appreciate what this issue is trying to accomplish.
